app/forms/decidim/kids/minor_account_form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    class MinorAccountForm < Decidim::Form
+      mimic :minor_account
+
+      attribute :name, String
+      attribute :birthday, Decidim::Attributes::LocalizedDate
+      attribute :email, String
+      attribute :password, String
+      attribute :password_confirmation, String
+      attribute :tos_agreement, Boolean
+
+      validates :name, presence: true
+      validates :birthday, presence: true
+      validates :email, presence: true, "valid_email_2/email": { disposable: true }
+      validates :password, presence: true, password: { name: :name, email: :email, username: :nickname }
+      validates :password_confirmation, presence: true, if: :password_present
+      validates :tos_agreement, acceptance: true
+
+      validate :valid_minor_age
+
+      private
+
+      def password_present
+        password.present?
+      end
+
+      def valid_minor_age
+        min_minor_age = Decidim::Kids.minimum_minor_age
+        minimum_adult_age = Decidim::Kids.minimum_adult_age
+
+        if birthday.present?
+          minor_age = ((Time.zone.now - birthday.to_time) / 1.year.seconds).floor
+          errors.add(:birthday, I18n.t("decidim.kids.minor_account.form.invalid_age")) unless minor_age.between?(min_minor_age, minimum_adult_age - 1)
```

Comment:
I think the `- 1` is not necessary.

```suggestion
          errors.add(:birthday, I18n.t("decidim.kids.minor_account.form.invalid_age")) unless minor_age.between?(min_minor_age, minimum_adult_age)
```

Reply:
Right, but conceptually, maybe we should rename this variable to `maximum_minor_age` or change the other places to add the -1. cos, if the minor's age is already the  `minimum_adult_age`, then it can create an account on his own right?

Reply:
done